src/en/space-station-14/roleplay-lore/proposals/Departmental-Operating-Procedure.md

```diff
@@ -0,0 +1,122 @@
+
+I will split these procedures between two categories 
+
+* Flavor Text
+  - Nanotrasen themed guides that offer more fun than actual advice, think of things like "Interogation 101" which will feature an interogation guide complete with responses to certain answers, pretty much useless in terms of advice but is a fun bit of lore that is distrubted to security teams across Nanotrasen stations
+* Guidelines
+    - Think of these as the original SOP, a set of straight forward points and guides to explain how a department is to be run, but do not limit roleplay and instead encourage more. With things like medical evaluations and alert procedures. These could be appended with 'guideline' or just left as is such as "Alert Procedure" instead of "Alert Procedure Guidelines". 
+
+## "Encourage Roleplay"
+
+I use this phrase a lot to describe the DOP but I don't really explain it. And therefore I will implement a set of guidelines for the guidelines 
+
+### The Guidelines for the Guidelines
+
+1. A guideline should encourage roleplay through either pro-active (i.e. "Interview surronding crew members for a crime report") or passive encouragment (i.e. "Firearms should not be openly carried by security during green alert")
+2. A guideline should never be an exorbiant length. Instead of a guideline covering multiple things, cover one thing well. 
+
+## How will this look per department? NOTE: This is my example. I think this could be achieved in many ways but this is how I would do it. 
+
+---
+
+## ENFORCMENT
+(This is in my opinion how things should be enforced) 
+Any commanding officer over a department can choose whether or not to enforce certain guidelines. This will allow for repeatable and consistent roleplay expectations that are easy to learn but still allow for some mallability. 
+
+This would be no different than an HOS utilizing team names for security. Something that is totally up to the commanding officer to utilize but many players are already aware of their function. (This is an example of passive encouragment, as it passively promotes more coherent rp for security officers) 
+
+### Command: 
+
+Command will follow the "Command Operating Procedure"(COP), it will feature things like a "Chain of Command", "Bolting Procedure", "Alert Procedure", and a "Commanding Officers Work Behavior Guideline"
+
+#### Examples for Command: 
+
+##### Chain of Command 
+
+```mermaid
+graph TD;
+    subgraph Standard Alerts
+    idG[Under Green Alert]-->idCC1[Central Command];
+    style idG fill:#5ce65c,stroke:#333,stroke-width:4px
+    idCC1-->idC1[Captain];
+    idC1-->idHOP1[Head Of Personnel];
+    idHOP1-->idG1[All head of staff are equal below HOP];
```

Comment:
This is conflicting with roundstart briefing. According to it, all heads respond to captain directly.

Reply:
That is true. I will have to fix that. 

I am fairly certain the old SOP also made HOP the second in command. 

I will work something out